pandapower/pypower/dIbr_dV.py

```diff
@@ -60,3 +60,33 @@ def dIbr_dV(branch, Yf, Yt, V):
         It = asarray( Yt * asmatrix(V).T ).flatten()
 
     return dIf_dVa, dIf_dVm, dIt_dVa, dIt_dVm, If, It
+
+def dIbr_dV_new(branch, Yf, Yt, V):
+    # Compute currents.
+    if issparse(Yf):
+        If = Yf * V
+        It = Yt * V
+    else:
+        If = asarray( Yf * asmatrix(V).T ).flatten()
+        It = asarray( Yt * asmatrix(V).T ).flatten()
+
+    vb = range(len(V))
+    diagV = sparse((V, (vb, vb)))
+    diagVnorm = sparse((V / abs(V), (vb, vb)))
```

Comment:
Here there should be try except block because division with zero can occur due to abs(V), it will help enormously if throughindices one could also find the pandapower element (in dataframe) which leads to this exception , for example NaNor zero g_us_per_km attribute for lines, etc

Reply:
A division by 0 when dividing by the voltage magnitude? The second part of the comment (looking at the pandapower elements that can bring a division by zero) is not something that has to be addressed in a function to calculate the Jacobian. There are other functions and routines in pandapower that do that.

Reply:
yes we have this many times for networks which have problems in attribute and is very central as then the algorithm does not work.

Reply:
I see, but then the problem in that case is with the network data and with its processing, not with this pull request or with this specific function, so I guess we should shift the discussion somewhere else.